package.json

```diff
@@ -1,16 +1,19 @@
 {
   "name": "openfoodnetwork",
-  "version": "1.7.1",
+  "version": "1.0.0",
```

Comment:
Can I ask why the version was set to 1.0.0?

Reply:
hey Renan, welcome to OFN!
the commit mesage explains:
https://github.com/openfoodfoundation/openfoodnetwork/pull/6049/commits/fe7d7b54d10258da884775ac12475e09572090ea

I havent tried to remove the version entry, maybe that would be better.